dokum/app/views/source/index.phtml

```diff
@@ -15,9 +16,9 @@ if (!is_array($sources ?? null)) {
         </tr>
         </thead>
         <tbody>
-<?php foreach ($sources as $key => $source) : ?> 
+<?php foreach ($sources as $key => $source) : ?>
```

Comment:
Don't remove the final space.

With this space, create a new line.
```
<tbody>
  <tr>
```
Without it join the lines.
`<tbody><tr>`